Gemfile

```diff
@@ -6,7 +6,7 @@ gemspec
 
 gem "puma"
 gem "sqlite3"
-gem "web-console"
+gem "web-console", group: :development
```

Comment:
In gems, the Gemfile is only for development dependencies, so this is kind of redundant.